### DISCUSSION TITLE
Process title

### DISCUSSION THREAD

**Discussion by WORMSS** [lwsjs/local-web-server#181] [Q&A]
Hello, I don't know if this is documented anywhere or if it's even possible, but I would like to be able to set the process.title from the config files? 

Is this "a thing" or is this something that it can't do?

```json
{
  "scripts": {
    "host": "ws -c lws-config.js"
  }
}
```
```js
module.exports = {
  directory: './dist',
  spa: 'index.html',
  port: 8082,
  verbose: true,
  title: 'Some really cool name that I can easily see'
};
```

(One I have faked, to get the idea)
![image](https://user-images.githubusercontent.com/1384537/211764783-89284c81-32d0-447c-9704-afb74096e252.png)

**Comment by 75lb** [comment]
Hi, this feature does not yet exist in `ws`.. Does setting `process.title` work as required for you on your platform? The feature is not commonly used as on Linux and macOS the title is limited to four characters.. see [docs](https://nodejs.org/api/process.html#processtitle)

**Reply by WORMSS** [reply]
Absolutely fine on Windows, as you can see above, I got it to write "Reverse Proxy"

**Reply by IgeNiaI** [reply]
I'm looking forward to this feature too. I don't really need it configurable, just something different from vanilla "node" name. The "ws" or "web-server" will do too.
It does support more than 4 characters on ubuntu for sure. The http-server package does set it's process name to "http-server" and it works fine.

**Reply by 75lb** [reply]
I will look into it. 👍   https://github.com/lwsjs/local-web-server/issues/186

**Comment by 75lb** [comment]
Implemented. If you reinstall then run `ws --help`, you should see `--title` in the list of options..

**Comment by WORMSS** [comment]
Lovely, I assume it's also available from the config file as "title" property? @75lb

**Comment by 75lb** [comment]
Yes, it is.. if not, let me know.. 

A few weeks ago, I rewrote lws from scratch - it's not ready yet but the new version is much better IMO.. thanks for your feedback and input over the years.. If you have any more requests or ideas for this tool - now is the time to post your thoughts..

**Reply by WORMSS** [reply]
![image](https://github.com/user-attachments/assets/84d1e93b-8be9-495d-8327-3c0787ae5aa8)
Works with config file..

![image](https://github.com/user-attachments/assets/3ad73e73-042e-46a0-9e7a-469e98fb3b45)
Windows is messing up a little bit here with the title. But this time I am using `npx` and the `--title "http => https redirect"` param But the is displaying correctly where it matters.. 
![image](https://github.com/user-attachments/assets/2ac55887-aafe-459c-87ab-cfdf7c8fcd67)
